### PR TITLE
Added new app for showcase: Deskbookers

### PR DESCRIPTION
**Pull request by emilioicai**

### CODE REVIEW

**Comment by mkonicek** [comment]
Nice! The app looks cool.

@facebook-github-bot shipit

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -631,6 +631,12 @@ var apps = [
       'http://richard-cao.github.io/2016/02/06/Reading-App-Write-In-React-Native/',
     ],
   },
+  {
+    name: 'Deskbookers',
+    icon: 'http://a4.mzstatic.com/eu/r30/Purple69/v4/be/61/7d/be617d63-88f5-5629-7ac0-bc2c9eb4802a/icon175x175.png',
+    linkAppStore: 'https://itunes.apple.com/nl/app/deskbookers/id964447401?mt=8',
```
Comment (chirag04) [inline website/src/react-native/showcase.js]: we should keep the list alphabetically sorted and use `link` key

**Comment by chirag04** [comment]
@emilioicai this merged but can you still update with those changes?

**Comment by mkonicek** [comment]
Good catch @chirag04, thank you! Will keep it in mind when reviewing.

**Pull request closed without merging**